hooks/post_gen_project.py

```diff
@@ -0,0 +1,21 @@
+from shutil import copyfile
+import os
+
+base_path = os.path.split(os.getcwd())[0]
+base_path_libs = os.path.join(base_path, 'libs')
+target_path_libs = os.path.join(os.getcwd(), 'libs')
+
+try:
+    alldir = os.listdir(base_path_libs)
+    libs = filter(lambda x: x.endswith('.jar'), alldir)
+except:
+    libs = []
```

Comment:
While this won't break, the `except` clause is too broad: it will hide any problems in the code inside of the `try` block.

I've just realized this will be much simpler if we just use the `glob` module, which is already portable and won't break if the directory doesn't exist:

```
import glob

for src in glob.glob('../libs/*.jar'):
    dest = os.path.join('libs', os.path.basename(src))
    copyfile(src, dest)
```

Reply:
I didn't know the glob module, thanks for this! =)